fs/pacer.go

```diff
@@ -70,20 +70,6 @@ func (p *Pacer) SetCalculator(c pacer.Calculator) {
 	p.Pacer.SetCalculator(c)
 }
 
-// ModifyCalculator calls the given function with the currently configured
-// Calculator and the Pacer lock held.
-func (p *Pacer) ModifyCalculator(f func(pacer.Calculator)) {
-	p.ModifyCalculator(func(c pacer.Calculator) {
```

Comment:
We could fix this according to my idea if you want

Reply:
Yes, I will do that!